java/src/jmri/jmrit/beantable/LightTableAction.java

```diff
@@ -1006,7 +1006,7 @@ void cancelPressed(ActionEvent e) {
         // remind to save, if Light was created or edited
         if (lightCreatedOrUpdated) {
             InstanceManager.getDefault(jmri.UserPreferencesManager.class).
-                    showInfoMessage("Reminder", "Remember to save your Light information.", getClassName(), "remindSaveLight");
+                    showInfoMessage(Bundle.getMessage("ReminderTitle"), Bundle.getMessage("ReminderSaveString", Bundle.getMessage("MenuItemLightTable")), getClassName(), "remindSaveLight");
```

Comment:
This needs a ```// NOI18N``` marker to indicate that these String literals do not need translation.